python/doc/examples/reliability_sensitivity/reliability/plot_multi_form.py

```diff
@@ -28,7 +28,7 @@
 graphPDF.setXTitle(r"$x_1$")
 graphPDF.setYTitle(r"$x_2$")
 graphPDF.setLegendPosition("lower right")
-view = otv.View(graphPDF)
+view = otv.View(graphPDF, contour_kw={"norm": "log"})
```

Comment:
old versions of matplotlib only accept an instance of matplotlib.colors.Normalize and not a string
(we require matplotlib>=3.x, here the test fails for 3.0.2 from debian 10)

eg the viewer must pass LogNorm class here instead, see here for other classes

https://matplotlib.org/3.3.4/api/colors_api.html

Reply:
OK, this should be good now.